pkg/kube/activity.go

```diff
@@ -161,8 +163,23 @@ func GenerateBuildNumber(activities typev1.PipelineActivityInterface, pipelines
 	return build, answer, nil
 }
 
+func lazyUpdateSourceRepositoryCRD(jxClient versioned.Interface, ns string, activity *v1.PipelineActivity) error {
```

Comment:
the 'laziness' is the way that this method is called, rather than anything to do with the method itself. I suggest dropping 'lazy' from the function name